rest/blip_api_no_race_test.go

```diff
@@ -76,8 +74,7 @@ func TestBlipPusherUpdateDatabase(t *testing.T) {
 		}()
 
 		// and wait for a few to be done before we proceed with updating database config underneath replication
-		_, err := rt.WaitForChanges(5, "/{{.keyspace}}/_changes", "", true)
-		require.NoError(t, err)
+		rt.WaitForChanges(5, "/{{.keyspace}}/_changes", "", true)
```

Comment:
I am not sure how this test is going to be reliable with the new exact length checks.

Does the test need changing, or a new helper that is tolerant to more changes than expected?

Reply:
Good catch, this _doesn't_ work correctly but I didn't run this test enough times to expose this problem.

Reply:
I changed this test but it is skipped which is why it didn't. The test doesn't work correctly even when it is unskipped, but that is covered in CBG-1908. I don't think this test is relevant anymore as written, and it doesn't work correctly after the BlipTester changes.